r/tests/testthat/test-python-flight.R

```diff
@@ -15,6 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+skip("Temporarily skip this")
```

Comment:
Remove this (also why?)

Reply:
Ah shoot, sorry, I wasn't supposed to commit that 😂